src/storage/index/LookupProcessor.h

```diff
@@ -13,15 +13,17 @@
 namespace nebula {
 namespace storage {
 
+extern ProcessorCounters kLookupCounters;
```

Comment:
Why not define all counters in `GraphStorageServiceHandler`?

Reply:
Each individual processor could have it own counters. So it makes more sense to let the processors own the counters